docker-compose.yml

```diff
@@ -98,10 +98,10 @@ services:
       - POSTGRES_PASSWORD
       - POSTGRES_DB=minion
     healthcheck:
-      test: ["CMD-SHELL", "pg_isready -U ${POSTGRES_USER}"]
-      interval: 1s
+      test: ["CMD-SHELL", "pg_isready -U $$POSTGRES_USER -d $$POSTGRES_DB"]
+      interval: 10s
```

Comment:
This check is used for other containers that wait for Postgres to be healthy. Suggest adding a start_interval of 1s so that the container reports healthy as soon as possible